response_operations_ui/controllers/party_controller.py

```diff
@@ -31,6 +31,15 @@ def get_party_by_ru_ref(ru_ref):
     return response.json()
 
 
+def try_get_party_id_by_ru_ref(ru_ref):
+    """Attempts to get party id from ru_ref, does not error if not found"""
```

Comment:
So what does it return if found?  This would benefit from sphinx style docstrings:
```
"""Calls party service to get the party_id of a business based on its ru_ref
:param ru_ref: An 11 digit string referring to a business
:returns: A party_id if the ru_ref is found.  An empty string otherwise.
:raises KeyError: Raised if the response from party is successful but doesn't contain an 'id' key.
"""
```

Reply:
I dont mind documenting when needed , but in this case the function name is pretty descriptive. 
Have modified the doc string to say returned value if not found